kive/container/tests.py

```diff
@@ -3151,7 +3187,9 @@ def test_full_argument_formatting(self):
         self.assertEqual(
             command_args,
             [
-                "--optional_input", "/mnt/input/optional_input", "--",
+                "--optional_input", "/mnt/input/optional_input",
+                "--multiple_optional_input*", "/mnt/input/multiple_optional_input0",
```

Comment:
I don't think that star should be there. That should only appear when defining the list of arguments in the Singularity file or displaying the list in Kive, not when actually building a command line.

Reply:
Ah! Yes, I agree. We've definitely discussed this, and I agree. Good catch, thanks.